modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -1,270 +1,306 @@
-Selected Page (Optional):: Set the page to open.
+API:: Specifies the API.
+Section (Optional):: Selects the section to open. If the *Selected page (optional)* setting is empty, the *Section (optional)* setting is ignored.
+Selected Page (Optional):: Selects the page to open.
 
 == API Header
 
-The API header component displays the API Header including icon, name, and description.
+The *API Header* component displays the API header, including icon, name, and description.
 
-API:: Choose the API to show.
+You can specify values for the following parameters of the *API Header*:
+
+API:: Specifies the API to show.
 
 == Application Analytics Graph
 
-The application analytics graph component displays statistics about the use of an application.
+The *Application Analytics Graph* component displays statistics about the use of an application, such as traffic, response time, and application health.
+
+You can specify values for the following parameters of the *Application Analytics Graph*:
 
-Application:: Choose the application to show.
+Application:: Specifies the application to show.
 
 == Application Details
 
-The application details component displays details about an application.
+The *Application Details* component displays details about an application.
 
 // check component description
 
-Application:: Choose the API to show.
+You can specify values for the following parameters of the *Application Details*:
+
+Application:: Specifies the API to show.
 
 == Application Listing
 
-The application listing component shows applications that use an API.
+The *Application Listing* component shows applications that use an API.
 
-New Application URL:: Set a new URL for the application.
-Card Margin:: Set the amount of margin space around the card.
-Card Border:: Set the thickness of the card borders.
-Card Border Radius:: Set the radius of the card borders. A larger radius makes the cards look more rounded.
-Card Shadow:: Set the shadow beneath the card.
-Card Width:: Set the card width.
-Card Height:: Set the card height.
+You can specify values for the following parameters of the *Application Listing*:
+
+New Application URL:: Specifies a new application URL.
+Card Margin:: Specifies the amount of margin space around the card.
+Card Border:: Specifies the card border thickness.
+Card Border Radius:: Specifies the card border radius. Cards with a larger radius look more rounded.
+Card Shadow:: Specifies the shadow beneath the card.
+Card Width:: Specifies the card width.
+Card Height:: Specifies the card height.
 
 // check *New Application URL* description
 
 == Learn More Button
 
-The learn more button component provides portal users with a custom page with additional information about the API.
+The *Learn More Button* component provides portal users with a custom page with additional information about the API.
+
+You can specify values for the following parameters of the *Learn More Button*:
```

Comment:
ditto

Reply:
ENDING EDIT HERE. This is all repetitive and my previous comments apply to all the rest.